build/ToolsetPackages/ClosedToolset.csproj

```diff
@@ -0,0 +1,13 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
It's holding all of the nuget packages we use in roslyn-internal.  It's a direct port of the project.json which did this.